### ISSUE TITLE
Has Breadcrumb public property in TypeScript?

### ISSUE THREAD

**Issue by ybiquitous** [bugsnag/bugsnag-js#372]
### Expected behavior

Following code will not produce compile errors:

```typescript
class Breadcrumb {
  constructor(public name: string, public metaData?: object, public type?: string, public timestamp?: string) {}
}

const bc = new Breadcrumb("foo")
bc.name
bc.metaData
bc.type
bc.timestamp
```

### Observed behavior

Above code produces compile error. Current code:

https://github.com/bugsnag/bugsnag-js/blob/2ac156b039a8d1b841151968acdc19f0da809dea/types/breadcrumb.d.ts#L2

https://github.com/bugsnag/bugsnag-js/blob/2ac156b039a8d1b841151968acdc19f0da809dea/base/breadcrumb.js#L4

### Steps to reproduce

```typescript
import bugsnag from "bugsnag-js"

export default bugsnag({
  beforeSend: report => {
    report.breadcrumbs.map(breadcrumb => breadcrumb.metaData)
    //=> Compile error: "Property 'metaData' does not exist on type 'Breadcrumb'. [2339]"
  }
})
```

### Version

4.7.2

### Additional information

None

**Comment by ybiquitous** [comment]
@bengourley Thanks for quick response! I'm expecting new release! 😄

**Comment by ybiquitous** [comment]
Thanks! 😄

**Comment by bengourley** [comment]
This is now published in v4.7.3 🚀
